packages/web-pkg/src/composables/piniaStores/webWorkers.ts

```diff
@@ -0,0 +1,52 @@
+import { defineStore } from 'pinia'
+import { Ref, ref, unref } from 'vue'
+import { v4 as uuidV4 } from 'uuid'
+import { UseWebWorkerReturn, useWebWorker } from '@vueuse/core'
+
+export interface WebWorker<T = any> extends UseWebWorkerReturn<T> {
+  id: string
+}
+
+export type WorkerTopics = 'startProcess' | 'tokenUpdate'
+
+export const useWebWorkersStore = defineStore('webWorkers', () => {
+  const workers = ref([]) as Ref<WebWorker[]>
+
+  const createWorker = <T = any>(workerUrl: string): WebWorker<T> => {
+    const result = useWebWorker(workerUrl, { type: 'module', name: uuidV4() })
+    const worker = { id: uuidV4(), ...result }
```

Comment:
Why don't you re-use the `name` as the id? Any benefit in having two different uuids for the `id` and `name` props? 🤔

Reply:
I just didn't see it because I added the name later I guess, good catch!